specs/altair/fork.md

```diff
@@ -17,7 +17,7 @@
 
 ## Introduction
 
-This document describes the process of moving from Phase 0 to Phase 1 of Ethereum 2.0.
+This document describes the process of the first upgrade of Ethereum 2.0: the Altair hardfork, introducing Light Client support.
```

Comment:
nitpicking: it's not only light client support. The scope is described in https://github.com/ethereum/eth2.0-specs/blob/c979e690030d22661b1d7e5c433ecf4a809683db/specs/altair/beacon-chain.md#introduction


```suggestion
This document describes the process of the first upgrade of Ethereum 2.0: the Altair hard fork, introducing light client support and other improvements.
```